src/main/java/com/npsproject/it4045npsproject/controller/NPSController.java

```diff
@@ -21,14 +24,25 @@ public NPSController(ParkService parkService, ParkSearchService parkSearchServic
         this.parkSearchService = parkSearchService;
     }
 
-    @GetMapping("/{name}")
+    /**
```

Comment:
Add jdoc to describe these new endpoints. 

https://www.baeldung.com/java-clean-code